addons/website_event/views/event_templates_page_registration.xml

```diff
@@ -124,8 +124,12 @@
     <div t-if="toast_message" class="o_wevent_register_toaster d-none" t-att-data-message="toast_message"/>
     <div t-if="not event.event_registrations_open" class="bg-white mb-5">
         <div class="alert alert-info mb-0 d-flex justify-content-between align-items-center" role="status">
-            <t t-if="event.start_sale_date and event.start_sale_date &gt; datetime.date.today()">
-                <em class="col-md-8">Ticket Sales starting on <t t-esc="event.start_sale_date"/></em>
+            <t t-if="not event.event_registrations_started">
+                <em class="col-md-8">Ticket Sales starting on
+                    <span class="" t-esc="event.start_sale_datetime"
+                            t-options="{'widget': 'datetime', 'tz_name': event.date_tz, 'format': 'MM/dd/yyyy hh:mm a'}"/>
```

Comment:
hardcoded format :/

Reply:
Indeed try to use long / short formats instead (with additional timezone display if necessary).

Reply:
changed to "short", thanks for the hint!